packages/client/lib/net/server/rlpxserver.ts

```diff
@@ -13,17 +13,28 @@ export interface RlpxServerOptions extends ServerOptions {
 
 const ignoredErrors = new RegExp(
   [
-    'EPIPE',
+    // Peer socket connection
     'ECONNRESET',
-    'ETIMEDOUT',
-    'NetworkId mismatch',
-    'Timeout error: ping',
+    'EPIPE', // (?)
+    'ETIMEDOUT', // (?)
+
+    // ETH status handling
     'Genesis block mismatch',
-    'Handshake timed out',
+    'NetworkId mismatch',
+    'Unknown fork hash',
+
+    // DPT message decoding
+    'Hash verification failed',
     'Invalid address buffer',
-    'Invalid MAC',
     'Invalid timestamp buffer',
-    'Hash verification failed',
+    'Invalid type',
+    'Timeout error: ping', // connection
+
+    // ECIES message encryption
+    'Invalid MAC',
+
+    // Client
+    'Handshake timed out', // Protocol handshake
```

Comment:
Not related to this PR but it would be nice to know why it is OK to ignore these errors.

Reply:
Can't answer for all cases (mainly not sure about the peer socket connection errors) but at least the very most of the other cases are either cases where the peer is sending malformed data ('Invalid MAC', 'Hash verification failed', all other DPT stuff (except time out)) for unknown reasons or we don't want to have certain data by protocol decision, 'NetworkId mismatch' as the most obvious case where we just don't want to connect to a peer which is on a different network.

All this is taking place under the assumption that our actual devp2p implementation along these kind of failures is correct. So we generally don't want to have these errors displayed again and again when they are just expectable. If they are occurring too often though, this might be an indicator that there is something wrong with our implementation. Sometimes hard to distinguish though. But it's likely a good idea to deactivate these filters temporarily from time to time and see what errors come through in what level of quantity.